apps/meteor/tests/e2e/admin-users.spec.ts

```diff
@@ -1,88 +1,184 @@
+		});
+
+		test('Login as a newly created user and verify its status is active', async () => {
+			await page.goto('/login');
+			await test.step('should log in as the newly created user', async () => {
+				await poRegistration.username.fill(user.data.username);
+				await poRegistration.inputPassword.fill(DEFAULT_USER_CREDENTIALS.password);
+				await poRegistration.btnLogin.click();
+			});
+
+			await test.step('Assert user is logged in', async () => {
+				await expect(poUtils.mainContent).toBeVisible();
+			});
+		});
+	});
+
+	test.describe('User Status Management', () => {
+		test.use({ storageState: Users.admin.state });
+
+		test.beforeEach('Go to /admin/users', async ({ page }) => {
+			admin = new Admin(page);
+			await page.goto('/admin/users');
+		});
+
+		test('After the first login, the user gets listed under the Active tab', async () => {
+			await admin.tabs.users.inputSearch.fill(user.data.username);
+
+			await test.step('is visible in the All tab', async () => {
+				await admin.tabs.users.tabActive.click();
+				await expect(admin.getUserRowByUsername(user.data.username)).toBeVisible();
+			});
+
+			await test.step('is not visible in the Pending tab', async () => {
+				await admin.tabs.users.tabPending.click();
+				await expect(admin.getUserRowByUsername(user.data.username)).not.toBeVisible();
 			});
-		},
-	);
+
+			await test.step('is visible in the Active tab', async () => {
+				await admin.tabs.users.tabActive.click();
+				await expect(admin.getUserRowByUsername(user.data.username)).toBeVisible();
+			});
+
+			await test.step('is not visible in the Deactivated tab', async () => {
+				await admin.tabs.users.tabDeactivated.click();
+				await expect(admin.getUserRowByUsername(user.data.username)).not.toBeVisible();
+			});
+		});
+	});
+
+	test.describe.serial('Admin Role Management', () => {
+		test.use({ storageState: Users.admin.state });
+
+		test.beforeEach('Go to /admin/users', async ({ page }) => {
+			admin = new Admin(page);
+			await page.goto('/admin/users');
+		});
+
+		test('Make a newly created user as admin', async () => {
+			await admin.tabs.users.inputSearch.fill(user.data.username);
+
+			await test.step('User is visible in the All tab', async () => {
+				await admin.tabs.users.tabAll.click();
+				await expect(admin.getUserRowByUsername(user.data.username)).toBeVisible();
+			});
+
+			await test.step('make a user admin', async () => {
+				await admin.tabs.users.openUserActionMenu(user.data.username);
+				await admin.tabs.users.menuItemMakeAdmin.click();
+				await expect(admin.tabs.users.toastMessage).toContainText('User is now an admin');
+			});
+
+			await test.step('verify user is admin', async () => {
+				await admin.tabs.users.getUserRowByUsername(user.data.username).click();
+				await expect(admin.tabs.users.openDialog).toBeVisible();
+				await expect(admin.tabs.users.openDialog).toContainText('Admin');
+			});
+		});
+
+		test('Remove role as admin', async () => {
```

Comment:
Will this be dependent on the previous test? Maybe you could have a separate user who is already admin to remove it